code/game/objects/structures/stool_bed_chair_nest/chairs.dm

```diff
@@ -59,14 +59,17 @@
 	set category = "Object"
 	set src in oview(1)
 
-	if(!usr || !isturf(usr.loc))
-		return
-	if(usr.stat || usr.restrained())
+	if(config.ghost_interaction)
+		dir = turn(dir, 90)
+		handle_rotation()
 		return
+	else

```

Comment:
This doesn't need to be behind an else, and shouldn't be, as the if statement above performs an early return.